Extension.php

```diff
@@ -38,7 +38,9 @@ public function initialize()
         $root = $this->app['config']->get('general/branding/path') . '/';
 
         // Admin menu
-        $this->addMenuOption('Label translations', $root . 'labels', 'fa:flag');
+        if (!isset($this->config['show_menu']) || !(isset($this->config['show_menu']) && $this->config['show_menu'] == false)) {
+            $this->addMenuOption('Label translations', $root . 'labels', 'fa:flag');
+        }
```

Comment:
Add `show_menu` to `getDeafultConfig()` and it will always be set… and don't use loose comparison on boolean values in PHP.

The whole thing can then be simplified to something like:
```php
if ($this->config['show_menu']) {
    $this->addMenuOption('Label translations', $root . 'labels', 'fa:flag');
}
```